bin/grab-kernels

```diff
@@ -35,11 +35,15 @@ debs2tar() {
     mkdir "$tdir/ex"
     for deb in "$@"; do
         debug 2 "extracting $deb"
-        dpkg -x "$deb" "$tdir/ex" || {
+        ar x "$deb" --output "$tdir/ex" || {
```

Comment:
we need to add `binutils` to system-setup. as it provides 'ar'. for completeness should probably add 'tar' there too.

Reply:
We have "build-essential" there which covers "binutils".

"tar" has "required" priority so it is present by default because "dpkg" depends on it.

Reply:
Heh. Ubuntu uses no compression for armhf kernels.